client/src/components/footer/links.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import './footer.scss';
+
+class Links extends React.Component {
```

Comment:
maybe footer links? Or change style sheets

Reply:
Links are incapsulated in footer, we discussed it